Lib/glyphsLib/builder/features.py

```diff
@@ -132,3 +134,14 @@ def _build_gdef(ufo):
                      (glyph, ' '.join(unicode(p) for p in sorted(caretPos))))
     lines.append('} GDEF;')
     return '\n'.join(lines)
+
+
+def replace_feature(tag, repl, features):
+    if not repl.endswith("\n"):
+        repl += "\n"
+    return re.sub(
+        r"(?<=^feature %(tag)s {\n)(.*)(?=^} %(tag)s;$)" % {"tag": tag},
```

Comment:
I think the middle `(.*)` should be made non-greedy to avoid consuming an unrelated feature, e.g. if the following is possible:

```
feature liga {
} liga;

feature calt {
} calt;

feature liga {
} liga;
```

(I don't know if it is possible)

Reply:
good point, but also bear in mind this is not meant to work with arbitrary UFO features.fea, but only those that where composed by glyphsLib ifself by concatenating the GSFont's features.

is it possible from the Glyphs.app's features panel to add multiple features with the same tag? I can't try right now.
If so, we need to support such case.

Reply:
also, it's not clear what the "Replace Feature" custom parameter should do in this case.
I made it only replace the first occurrence of a feature block with a given tag (count=1).